docs/admin/deployment.md

```diff
@@ -365,10 +370,15 @@ SCM format required on instance 0
 SCM format required on instance 1
 ```
 
-To format the storage and start the engine processes, we run the following on a separate
+The `daos_server start --auto-format` option has now been added to allow automatic formatting on
+startup in the event that externally triggered format is not desirable. When using this option
+the storage format will occur on `daos_server` startup without any user intervention assuming the
+supplied config file is valid.
+
+To manually format the storage and start the engine processes, we run the following on a separate
 terminal window and verify that engine processes (ranks) have registered with the system.
-Note the subsequent system query command may not show ranks started immediately after the
-storage format command returns so leave a short period before invoking.
+Note the subsequent system query command may not show ranks started immediately after the storage
+format command returns so it is recommended to leave a short delay before invoking.
```

Comment:
Nit, a minimal value or a rough value of delay could be helpful for the end user.

Reply:
fixed